client/src/components/ItemDetail/ItemDetail.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import ItemDetail from "./ItemDetail";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("ItemDetail Component Tests", () => {
```

Comment:
Could you add test for selectActionCallBack function?

Reply:
Tests for this `selectActionCallback` added to `ItemCard.test.tsx` and `ItemCardCompact.test.tsx`.
Added test for closeCallback in `ItemDetail.test.tsx`
Refactored how items are updated in #203, so will leave additional test for changing items to that PR